src/coreclr/src/vm/runtimehandles.cpp

```diff
@@ -2919,8 +2919,6 @@ void QCALLTYPE ModuleHandle::ResolveType(QCall::ModuleHandle pModule, INT32 tkTy
 
     BEGIN_QCALL;
 
-    _ASSERTE(!IsNilToken(tkType));
```

Comment:
It would be useful to add test coverage for the nil tokens

Reply:
Like this? https://github.com/dotnet/runtime/pull/45292/commits/69d94952f1206fe3bdcb055c5d2589f734519101

Reply:
Yes. I would also add the nil tokens with type token specified:
```
0x01000000 // mdTypeRefNil
0x02000000 // mdTypeDefNil
0x1b000000 // mdTypeSpecNil
```
And similar for the other ones: https://github.com/dotnet/runtime/blob/master/src/coreclr/src/inc/corhdr.h#L1524

Reply:
Added